### PR TITLE
docs(deepsegmenter): update readme and manifest

### PR DESCRIPTION
**Pull request by Yongxuanzhang**
This pr is for JINAHUBV1-371. 
Let me know what else should we add to the content. 😄 

In the Manifest file:
- [x] short description at least includes the domain space 
- [x] the “documentation” points to the README (URL) of the respective Image
- [x]  “Keywords” include; audio, image, nlp, video

In the README:
- [x] markdown
- [x] 1-2 sentence description of the "Executor" and the functionality of it
- [x] Make sure we have a Use (Example Section) with code snipped of  "how to use it in flow"

### CODE REVIEW

**Review by theUnkownName — APPROVED** [review]

File: segmenters/nlp/DeepSegmenter/README.md [hunk 1]
```diff
@@ -1,3 +1,55 @@
...
 
-Designed with ASR outputs in mind, DeepSegment uses BiLSTM + CRF for automatic sentence boundary detection. It significantly outperforms the standard libraries (spacy, nltk, corenlp ..) on imperfect text and performs similarly for perfectly punctuated text. 
\ No newline at end of file
+Designed with ASR outputs in mind, [DeepSegment](https://bpraneeth.com/projects/deepsegment) uses BiLSTM + CRF for automatic sentence boundary detection. It significantly outperforms the standard libraries (spacy, nltk, corenlp ..) on imperfect text and performs similarly for perfectly punctuated text. 
+
+Additional links:
+- [DeepSegment](https://bpraneeth.com/projects/deepsegment)
+- [BiLSTM-CRF paper](https://arxiv.org/abs/1508.01991)
+
+##Usage:
```
Comment (theUnkownName) [inline segmenters/nlp/DeepSegmenter/README.md:9]: Leave a space between the hashtags and the title for correct markdown :)

File: segmenters/nlp/DeepSegmenter/manifest.yml [hunk 2]
```diff
@@ -9,7 +9,7 @@ description: Designed with ASR outputs in mind, DeepSegment uses BiLSTM + CRF fo
 author: Jina AI Dev-Team (dev-team@jina.ai)
 url: https://jina.ai
 vendor: Jina AI Limited
-documentation: https://github.com/jina-ai/jina-hub
+documentation: https://github.com/jina-ai/jina-hub/blob/master/segmenters/nlp/DeepSegmenter/README.md
 version: 0.0.9
```
Comment (cristianmtr) [inline segmenters/nlp/DeepSegmenter/manifest.yml:13]: Bump version

**Review by cristianmtr — CHANGES_REQUESTED** [review]
bump version

File: segmenters/nlp/DeepSegmenter/README.md (hunk 1 above)
Comment (Yongxuanzhang) [reply segmenters/nlp/DeepSegmenter/README.md]: ✅

File: segmenters/nlp/DeepSegmenter/manifest.yml (hunk 2 above)
Comment (Yongxuanzhang) [reply segmenters/nlp/DeepSegmenter/manifest.yml]: ✅

**Review by cristianmtr — APPROVED** [review]

**Pull request merged**